src/store/index.js

```diff
@@ -0,0 +1,28 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import { composeWithDevTools } from 'redux-devtools-extension/logOnlyInProduction';
+
+import rootReducer from './reducers';
+import subscribe from './subscribers';
+
+let store = null;
+export const initStore = preloadedState => {
+  const composeEnhancers = composeWithDevTools({
+    // Specify extension’s options like name,
+    //  actionsBlacklist, actionsCreators, serialize...
+  });
+
+  const enhancer = composeEnhancers(
+    applyMiddleware(thunk),
```

Comment:
@jberube If we only make simple use of `thunk`, I believe we could remove the dependency and write it as:
```
const thunk = store => {
  const dispatch = store.dispatch
  const getState = store.getState
  return next => action => {
    if (typeof action === 'function') {
      return action(dispatch, getState)
    }
    return next(action)
  }
}
```

Reply:
I would rather not, for maintenance purpose. Or if we start using TypeScript.

Reply:
@jberube If we start using TypeScript, let us have a discussion first. I don't want us to start plugging it everywhere, but rather have it in useful places such as validation structs, etc.